nailgun/entities.py

```diff
@@ -6361,7 +6361,11 @@ def __init__(self, server_config=None, **kwargs):
             'network': entity_fields.IPAddressField(required=True),
             'to': entity_fields.IPAddressField(),
             'vlanid': entity_fields.StringField(),
+            'mtu': entity_fields.IntegerField(),
         }
+
+        if _get_version(server_config) < Version('6.4'):
```

Comment:
I wouldn't bother with this if I was you, the Ansible modules don't set the `version` attr in `server_config`. Plus we have different branches for different supported versions see https://github.com/SatelliteQE/nailgun/branches. Kind of like we 2 strategies for supporting different server versions.

Someone else may be able to give an authoritive answer on what server versions the master branch should support

Reply:
Yes, we have a 6.4 branch, if needed we can backport this to 6.4 later. Currently, Master points to 6.5 or foreman 1.20. 
we may soon have documentation published around this, to have a clear idea